GOVERNANCE.md

```diff
@@ -65,17 +79,92 @@ The Chairs are expected to perform a variety of functions to support Oversight C
+
+### Candidate Eligibility
+
+Key Eligibility Requirements:
+
+* **Required**: The candidate must satisfy at least one of the below criteria to be eligibile to stand for election for oversight committee membership.
+  * current oversight commitee member, or
+  * current maintainer of any **core** project, or
+  * any eligible voter with endorsement from two other eligible voters from different companies/organizations (one endorsement can be from the same organization as the candidate's organization). **Note:** Eligibility may be updated in future to ensure at least one of the endorsements is from a maintainer of the **core** project.
+* Technical Expertise: Candidates should be senior, respected technical leaders or contributors with significant experience relevant to the project's domain.
+* Demonstrated Commitment: They must have the time and willingness to actively participate in meetings, discussions, and the work of the committee.
+* Neutrality and Project-First Mindset: A crucial requirement is the ability to act as a neutral party, prioritizing the good of the project over any personal or company interests.
+* Good Standing: Candidates should have a clean record with respect to the project's Code of Conduct. The CNCF, for example, requires that candidates for its Code of Conduct Committee have no violations in the past 18 months.
+
+### Election Process
+
+The election process is as follows:
+
+1. **Call for Nominations**: One month before the election, a call for nominations will be sent out to the OSCAL Compass community.
+2. **Nomination Period**: Community members will have one month to nominate candidates for the Oversight Committee. Self-nominations are permitted.
+3. **Candidate Statements**: Each candidate will be asked to provide a statement of interest and qualifications.
+4. **Vetting**: There is usually a process for the existing committee or another body to review the nominations and confirm that the candidates meet the eligibility criteria.
+5. **Voting Period**: A two-week voting period will be held.
+6. **Results**: The results of the election will be announced within one week of the close of voting.
+7. **Voting Process and Tie-Breaking**: Elections will be held using an online preference election system which supports [Condorcet](https://en.wikipedia.org/wiki/Condorcet_method) elections, using Schulze completion rule.
+
+### Terms and Election Cycles
+
+Oversight committee members are elected to serve a two year term. Members can serve two consecutive terms (4 years). Bootstrap and terms that result in equal to or less than one year served are exempt.
+
+Election cycles are scheduled such that roughly half of the seats come up for re-election each year for purposes of continuity. The exact number of seats alternates between 3 and 4.
```

Comment:
Further to the point above, I think you should specify that the election cycle should be annual. it's not explicitly spelled out.